packages/utilities/getFirstFocusableChildNode.ts

```diff
@@ -0,0 +1,29 @@
+// https://github.com/theKashey/focus-lock/blob/master/src/utils/tabbables.js
+const focusableNodeSelectors = [
+  "button:enabled:not([readonly])",
+  "select:enabled:not([readonly])",
+  "textarea:enabled:not([readonly])",
+  "input:enabled:not([readonly])",
+
+  "a[href]",
+  "area[href]",
+
+  "iframe",
+  "object",
+  "embed",
+
+  "[tabindex]",
+  "[contenteditable]",
+  "[autofocus]"
+];
```

Comment:
You can make this more efficient by just calling .join(",") here once instead of twice each time getFirstFocusableChildNode is called.